.gitignore

```
@@ -1,5 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+calyptia-vivo-components*.tgz
```

Probably just *.tgz - we should never have those!
```suggestion
*.tgz
*.tar*
```